README.md

```diff
@@ -254,6 +254,32 @@ in `.baseline/copyright/*.txt`. The largest file (sorted lexicographically) will
 
 To automatically update all files with mismatching/missing copyrights, run `./gradlew format`.
 
+### Inclusive Language 
```

Comment:
Thoughts on moving this to fit into https://github.com/palantir/gradle-baseline/tree/develop/docs/best-practices/java-coding-guidelines
It's not java-specific, but otherwise I think that's a reasonable place for guidelines to live (where this won't describe a specific check, but rather our guidance which will future checks may enforce)

Reply:
A new file in that directory that isn't specific to java could also work nicely, though a bit harder for folks to discover. I defer to your judgement :-]

Reply:
good idea! moved them to a new file in that directory and linked from the best practices readme.

Reply:
i'll update the top-level readme with information about the checkstyle enforcement in that PR.